docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -1,20 +1,20 @@
 Developer Setup
 ===============
 
-Creating a ``bash`` development shell
+Creating a ``bash`` Development Shell
```

Comment:
I've also added a guideline on the _Documentation_ page about capitalization in section headings, and have tried to update everything to follow that rule.